### ISSUE TITLE
WooPayments makes the Analytics text too thin, difficult to read

### ISSUE THREAD

**Issue by Aljullu** [Automattic/woocommerce-payments#9572]
### Describe the bug

Installing WooPayments adds a line of CSS that makes text inside WooCommerce Analytics tables very thin (`.woocommerce-table { font-weight: 300; }`).

### To Reproduce

1. Install WooCommerce and WooPayments.
2. Go to Analytics > Products (for example).
3. Notice the weight of the text inside the table:

WooCommerce default | With WooPayments active
--- | ---
![Image](https://github.com/user-attachments/assets/f5b28839-708f-4654-bd26-740d96d1f7f6) | ![Image](https://github.com/user-attachments/assets/d0ac3cf5-9fee-447f-acf4-fb53456c7934)

### Actual behavior

Text has a font-weight of 300, making it difficult to read.

### Expected behavior

WooPayments shouldn't modify the styles of WooCommerce Analytics, and the font-weight should be kept at 400 (the default).

### Desktop (please complete the following information):

* OS: Fedora Linux
* Browser Firefox
* Version 133a1

### Additional context

* WooCommerce: 9.4.0-beta.2 and `trunk`
* WooCommerce Payments: 8.3.1

**Comment by c-shultz** [comment]
Thanks for the report @Aljullu !

I'm routing this to the disputes focus area with Transact because [the code that I believe is related was introduced for the disputes table](https://github.com/Automattic/woocommerce-payments/pull/453/files#diff-a4c6117e9085ecc98b0a1c4ffdb74ac0d7479d93bd1c495ef2a2afa2cf69a241R46).

**Comment by haszari** [comment]
Thanks @Aljullu  @c-shultz ! Marking as `high` since we should not be breaking upstream/core styles accidentally, and this could affect accessibility.
